lib/format/duration/formatters/default.ex

```diff
@@ -74,7 +78,7 @@ defmodule Timex.Format.Duration.Formatters.Default do
   defp format_component({:seconds, s}, str), do: str <> "#{s}S"
 
   defp deconstruct(%Duration{microseconds: micro} = duration),
-    do: deconstruct({Duration.to_seconds(duration, truncate: true), micro}, [])
+    do: deconstruct({Duration.to_seconds(duration, truncate: true), rem(micro, 1_000_000)}, [])
```

Comment:
Bug happens because `Duration.to_seconds` already includes microseconds in calculation so when microseconds amount exceeds 1 second the quotient part of microseconds is counted twice.